File: done/1893.py
# 1893 - Moon Phases
first, second = map(int, input().split())
if second <= 2:
    print('nova')
elif second <= 96:
    if first > second:
        print('minguante')
    else:
        print('crescente')
else:
    print('cheia')

File: done/1893.c
#include <stdio.h>

int main(void)
{
    int a, b;
    scanf("%d%d", &a, &b);

    if (b >= 0 && b <= 2)
    {
        printf("nova\n");
    }
    else if (b >= 97 && b <= 100)
    {
        printf("cheia\n");
    }

    if (b <= 96 && b >= 3)
    {
        if (a <= b)
        {
            printf("crescente\n");
        }
        else
        {
            printf("minguante\n");
        }
    }

    return 0;
}
